src/lib/y2storage/planned_volumes_list.rb

```diff
@@ -66,16 +67,32 @@ def deep_dup
     # its minimum size is taken instead. This gives a more useful sum in the
     # very common case that any volume has an 'unlimited' desired size.
     #
+    # If the optional argument "rounding" is used, the size of every volume will
+    # be rounded up. # @see DiskSize#ceil
+    #
+    # @param rounding [DiskSize, nil]
     # @return [DiskSize] sum of desired/min sizes in @volumes
-    def target_disk_size
-      @volumes.reduce(DiskSize.zero) { |sum, vol| sum + vol.min_valid_disk_size(target) }
+    def target_disk_size(rounding: nil)
+      @volumes.reduce(DiskSize.zero) do |sum, vol|
+        vol_size = vol.min_valid_disk_size(target)
+        vol_size = vol_size.ceil(rounding) if rounding
+        sum + vol_size
+      end
     end
 
     # Total sum of all current max sizes of volumes
     #
+    # If the optional argument "rounding" is used, the size of every volume will
+    # be rounded up. # @see DiskSize#ceil
+    #
+    # @param rounding [DiskSize, nil]
     # @return [DiskSize]
-    def max_disk_size
-      @volumes.reduce(DiskSize.zero) { |sum, vol| sum + vol.max_disk_size }
```

Comment:
This can be kept simple `{|sum, vol| sum + vol.max_disk_size.ceil(rounding) }` if you use a default `rounding` that returns the input. That is, `def ...(rounding: 1)` should do.